protocol/app/msgs/internal_msgs.go

```diff
@@ -52,10 +56,8 @@ var (
 		"/cosmos.crisis.v1beta1.MsgUpdateParamsResponse": nil,
 
 		// distribution
-		"/cosmos.distribution.v1beta1.MsgCommunityPoolSpend":         &distribution.MsgCommunityPoolSpend{},
```

Comment:
Blocking Comment: why are we moving this from internal to normal? Based on [this comment](https://github.com/cosmos/cosmos-sdk/blob/b729e981f1302d2490244b32aba7874a490d80ed/proto/cosmos/distribution/v1beta1/tx.proto#L45-L51), unless we adopt the new `x/protocolpool` module, this msg should still be executed only by `gov`. Therefore, this should stay as internal.

Reply:
I have been moving things back and forth too many times and moved this one by mistake.